.github/workflows/test-environment.yml

```diff
@@ -324,7 +324,7 @@ jobs:
         id: cspm-gcp-agent
         working-directory: deploy/deployment-manager
         run: |
-          . ./set_env.sh && ./deploy.sh && gcloud deployment-manager deployments update "${DEPLOYMENT_NAME}" --update-labels "${GCP_DEFAULT_TAGS}"
+          . ./set_env.sh && ./deploy.sh && gcloud compute instances update "${DEPLOYMENT_NAME}" --update-labels "${GCP_DEFAULT_TAGS}" --zone=us-central1-a
```

Comment:
It might be better to use the zone from environment variables. For example, you can define a global environment variable like this:
`AZURE_ZONE: "us-central1-a"`

Then, in your script, you can use the environment variable as follows:
`
. ./set_env.sh && ./deploy.sh && gcloud compute instances update "${DEPLOYMENT_NAME}" --update-labels "${GCP_DEFAULT_TAGS}" --zone="${AZURE_ZONE}"
`